go/vt/vtctld/api.go

```diff
@@ -60,8 +61,16 @@ const (
 	jsonContentType = "application/json; charset=utf-8"
 )
 
-// TabletWithURL wraps topo.Tablet and adds a URL property.
-type TabletWithURL struct {
+// TabletStats represents realtime stats from a discovery.LegacyTabletStats struct.
+type TabletStats struct {
```

Comment:
nit: does this struct need to be exported? It seems like it is used only within the package so it could be `tabletStats`.

Reply:
@deepthi it would be nice to be able to use these structs in the clients calling this API, but it's possible to maintain a copy if that's not preferred

Reply:
That is fine then. Just keep in mind that this is not an official API so it could change without notice.

Reply:
@deepthi sounds good 👍, thanks